lib/web_ui/lib/src/engine/layers.dart

```diff
@@ -543,12 +543,23 @@ class PlatformViewStyling {
 sealed class PlatformViewClip {
   PlatformViewClip positioned(PlatformViewPosition position);
 
-  ui.Rect get boundingRect;
+  /// The largest rectangle that is entirely inside the clip region. All
+  /// inside of this region is unclipped.
+  ui.Rect get innerRect;
 
-  bool covers(ui.Rect rect);
+  /// The bounding rectangle of the clip region. All content outside of this
+  /// region is clipped.
+  ui.Rect get outerRect;
 
   ScenePath get toPath;
 
+  static bool rectCovers(ui.Rect covering, ui.Rect covered) {
```

Comment:
https://api.flutter.dev/flutter/dart-ui/Rect/contains.html ?

Reply:
er https://api.flutter.dev/flutter/dart-ui/Rect/overlaps.html ?

Reply:
It is neither of those. `contains` acts on a single point. `overlaps` is that they have a non zero intersection, not that one completely covers the other (which is what we're checking).

Reply:
ack. thanks for entertaining my silly